src/System Application/App/Azure File Services API/src/Helper/AFSFormatHelper.Codeunit.al

```diff
@@ -0,0 +1,80 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Azure.Storage.Files;
+
+using System.Utilities;
+
+codeunit 8955 "AFS Format Helper"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    [NonDebuggable]
+    procedure AppendToUri(var UriText: Text; ParameterIdentifier: Text; ParameterValue: Text)
+    var
+        UriBuilder: Codeunit "Uri Builder";
+        Uri: Codeunit Uri;
+    begin
+        UriBuilder.Init(UriText);
+        if ParameterIdentifier <> '' then
+            UriBuilder.AddQueryParameter(ParameterIdentifier, ParameterValue)
+        else
+            UriBuilder.AddQueryFlag(ParameterValue);
+        UriBuilder.GetUri(Uri);
+        UriText := Uri.GetAbsoluteUri();
+    end;
+
+    [NonDebuggable]
+    procedure RemoveCurlyBracketsFromString("Value": Text): Text
+    begin
+        exit(DelChr("Value", '=', '{}'));
+    end;
+
+    procedure ConvertToDateTime(PropertyValue: Text): DateTime
```

Comment:
How will that DateTime Method will work in different languages?

What is the format of DateTime Values, that are returned by the API?
I couldn't find the API Documentation that fast.

Reply:
This should be language agnostic since it should only be called on the values from the API.
The function you mentioned is used in List Directories calls, and documentation (https://learn.microsoft.com/en-us/rest/api/storageservices/list-directories-and-files#datetime-format-and-api-version-for-timestamp-fields) says it uses the following formats:
ISO8601
RFC1123

In my tests they were evaluated correctly.